main.js

```diff
@@ -0,0 +1,109 @@
+let level = 0
+let cleanTimer
+const colors = ['#ee3e3e', '#3e52f0', '#ef3ef0', '#f0ed3e', '#1e945f']
+const width = $('.main').width() 
+const height = $('.main').height() 
+
```

Comment:
You should separate file and concepts, especially when using MVC, write your render functions in a different place